src/buzzer.c

```diff
@@ -26,32 +26,34 @@ BuzzFreq_t requested_buzz_freq;
 uint8_t requested_buzz_volume;
 int32_t requested_buzz_length;
 
-// The buzzer is hooked up to PWM5 output, but contains an oscillator of its own so volume and freq are ignored for now. :(
+/*
+ * The buzzer is hooked up to PWM5 output, but contains an oscillator of
+ * its own so volume and freq are ignored for now. :(
```

Comment:
Hmm yeah, that works. This code was ported from completely different hardware we have used in another project, I wanted to keep the same interface to minimize any code reuse issues. But you're right - it's not going to be used here.